WordPress/Tests/NamingConventions/PrefixAllGlobalsUnitTest.php

```diff
@@ -67,18 +61,7 @@ public function getErrorList( $testFile = 'PrefixAllGlobalsUnitTest.inc' ) {
 
 			case 'PrefixAllGlobalsUnitTest.1.inc':
```

Comment:
This case could simply fall through to the `default`. No need for a separate `return array()`.